lib/npg_pipeline/product/release/irods.pm

```diff
@@ -194,10 +197,60 @@ sub is_for_irods_release {
   return $enable;
 }
 
-sub _siblings_are_for_irods_release {
+=head2 is_for_pp_irods_release
+
+Return true if the portable pipeline product is to be released via iRODS,
+false otherwise.
+
+  $obj->is_for_pp_irods_release($product)
+
+=cut
+
+sub is_for_pp_irods_release {
+  my ($self, $product) = @_;
+  return $self->_is_for_release($product, $IRODS_PP_CONF_KEY);
+}
+
+=head2 glob_filters4publisher
+
+Returns a hash with glob filters for
+L<iRODS Tree Publisher|https://github.com/wtsi-npg/npg_irods/blob/master/bin/npg_publish_tree.pl>,
+which might be specified in the study configuration in the 'pp_irods' section.
+If the 'filters' key is present, the 'include' filter should be present. The
+'exclude' filter is optional.
+
+The format for both filters is validated. It is expected that an array of
+filter expressions is present.
+
+  my $filters = $self->glob_filters4publisher($product)
+
+=cut
+
+sub glob_filters4publisher {
```

Comment:
I'm loath to point out that this can now be tested independently.

Reply:
Agree